source/content/index.md

```diff
@@ -1,7 +1,30 @@
 ---
 title: Vogelfrei
 ---
-Contents of Vogelfrei game:
+## What is Vogelfrei?
+
+>
+> *For some, exile is the only fate. To be **vogelfrei** is to be cast out, beyond the protection of kings and gods alike, free in the most terrible way: free to be hunted, betrayed, and forgotten.*
+> 
+> *Yet even outlaws have a chance to carve their own destiny. A knight may fall from grace and take up the life of a mercenary. A peasant may flee the fields and become a bandit lord. A scholar may seek forbidden knowledge, risking soul and sanity. Steel and wit are a person’s greatest allies, for dangers lurk in the wilds and in the depths of the unknown. In this place, you are either bound by the laws of men—or cast out, left to forge your own destiny.*
+
+**Vogelfrei** is an OSR tabletop role-playing game set in a world shaped by feudal and early Renaissance societies. Players create characters from diverse social backgrounds — peasants, academics, warriors, rangers and rogues — where social status heavily influences life in civilized settlements. However, in untamed lands and forgotten ruins, the established order begins to crumble. This game emphasizes exploration across vast wildernesses — forests, mountains, and wastelands — as well as deep dungeons and the remains of lost cities.
+
+Wealth and survival are the primary goals. Characters are expected to seek riches by taking great risks, knowing that death is a constant possibility. Combat is not the main focus of the game; it is deadly and unpredictable. When fight do occur, it should be approached like a war — something to be won through tactics or avoided when the odds are against you.
```

Comment:
Combat is not the main focus of the game; it is deadly and unpredictable
-> Deadly and unpredictable, combat is not the universal solution to all problems (or something)
fight do -> fight does
approached like -> treated like